rosario_angelica/country.js

```diff
@@ -0,0 +1,64 @@
+const countryInput = document.getElementById('country-input');
+const searchBtn = document.getElementById('search-btn');
+const countryDetails = document.getElementById('country-details');
+const regionCountries = document.getElementById('region-countries');
+
+// Constants for API endpoints
+const BASE_URL = 'https://restcountries.com/v3.1';
+
+searchBtn.addEventListener('click', searchCountry);
+
+// Function to search for a country by name
+async function searchCountry() {
+  const countryName = countryInput.value.trim();
+  if (countryName) {
+    try {
+      const response = await fetch(`${BASE_URL}/name/${countryName}`);
+      const data = await response.json();
+
+      if (data.length > 0) {
+        const country = data[0];
+        const selectedDetails = [
+          { label: 'Flag', value: `<img src="${country.flags.png}" alt="${country.name.common} Flag" width="100">` },
```

Comment:
If possible, modify the width of image of flag using external css files (.css file), all design modification should be in css.You could do this by adding new file for example **country.css** and put your styling like below or depends o n your likings:
img {
width: 100 px;
}

This is stated in coding guidelines here: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR15hTWkgykAvEQujS74FHgJUa9UxiKSLF6C5MzuxYQZct0Vu7jQYWUyQAI_aem_AfbRDN52Fnw7TRXXR4uiXndLIWpv01TwxZMo7TvdPUV2YlzDnVgyePFxmQR_sexUJd0GnHMcLQM2qJdkeo00i445#slide=id.g286f540102c_1_0

Reply:
Additionally, **lines 22, 27, 47 and 59** exceeds **80 columns**, kindly fix it. Thank you.